lms/djangoapps/ccx/views.py

```diff
@@ -21,11 +20,14 @@
 from django.shortcuts import redirect
 from django.urls import reverse
 from django.utils.translation import gettext as _
+from django.utils.timezone import datetime, timezone
```

Comment:
Now I understand, in that case I would prefer to extend rather to edit. I think you can still use `import datetime` and use it like `datetime.datetime.now(pytz.UTC)`, which I also don't like but at least makes your PR 2 lines smaller.

In this case `datetime` is only used once by that variable, but you might encounter files where the usage of an import is huge and this kind of replacement could cause conflict if it is not double checked. @JuanDavidBuitrago 
```suggestion
```

Reply:
Done